membership-attribute-service/conf/riff-raff.yaml

```diff
@@ -1,6 +1,11 @@
 stacks: [membership]
 regions: [eu-west-1]
 deployments:
+  cloudformation:
+    type: cloud-formation
+    app: membership-attribute-service
+    parameters:
+      templatePath: cloudformation/membership-attribute-service.yaml
   ami:
```

Comment:
You will need to combine the AMI update with the Cloudformation step, or you'll get weird behaviour (see https://github.com/guardian/manage-frontend/pull/72).